src/shared/constants/status.js

```diff
@@ -126,13 +118,13 @@ export const LANGUAGES = [
 
 export const DISCIPLINES = [
   'Web Developer',
-  'Web Designer',
-  'Front-End Developer',
-  'Back-End Developer',
+  'UX/UI Designer',
+  'Frontend Developer',
```

Comment:
noooooooo `front-end` is the hill that I die on.

Also, I think `UX/UI` is more confusing than `Web Designer`.

Reply:
I'm less married to this one

Reply:
We can make this Digital Designer possibly.